modules/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2023-03-01
```

Comment:
rust-toolchain in the root dir applies to subdirectories, there is no need to add it here.